src/main/java/com/hillel/items_exchange/controller/AuthController.java

```diff
@@ -68,19 +80,20 @@
     }
 
     @PostMapping("/register")
-    public ResponseEntity<String> registerUser(@RequestBody @Valid UserRegistrationDto userRegistrationDto, BindingResult bindingResult) {
+    public ResponseEntity<Boolean> registerUser(@RequestBody @Valid UserRegistrationDto userRegistrationDto,
+                                                BindingResult bindingResult) {
+
         userRegistrationDtoValidator.validate(userRegistrationDto, bindingResult);
         if (bindingResult.hasErrors()) {
             throw new ResourceNotFoundException(bindingResult.toString());
         }
 
         Role role = roleRepository.findById(1L).orElseThrow(RoleNotFoundException::new);
```

Comment:
I think using repository into controller is not a good idea. We have the servise layer for it.

Reply:
Ok